esmvaltool/cmorizers/obs/cmorize_obs_piomas.py

```diff
@@ -0,0 +1,175 @@
+"""ESMValTool CMORizer for WOA data.
+
+Tier
+   Tier 2: other freely-available dataset.
+
+Source
+   http://psc.apl.uw.edu/research/projects/arctic-sea-ice-volume-anomaly/data/model_grid
+
+Last access
+   20190510
+
+Download and processing instructions
+   Download the desired files from:
+   https://pscfiles.apl.washington.edu/zhang/PIOMAS/data/
```

Comment:
Please specify the exact path (there are two versions and several subdirectories)

Reply:
Ping

Reply:
Done